src/PKSim.Presentation/Services/CompoundTask.cs

```diff
@@ -37,42 +37,6 @@ public class CompoundTask : BuildingBlockTask<Compound>, ICompoundTask
 
       public override Compound AddToProject() => AddToProject<ICreateCompoundPresenter>();
 
-      protected override IReadOnlyList<Compound> LoadFromTemplate(PKSimBuildingBlockType buildingBlockType)
-      {
-         var loadedCompounds = base.LoadFromTemplate(buildingBlockType);
-         if (loadedCompounds.Count <= 1)
-            return loadedCompounds;
-
-         //one compound and one or more metabolites
-         warnUserIfMetabolitesDoNotMatch(loadedCompounds);
-
-         return loadedCompounds;
-      }
-
-      private void warnUserIfMetabolitesDoNotMatch(IReadOnlyList<Compound> loadedCompounds)
```

Comment:
There is no easy way to actually check that. This was only required when importing a metabolite that already existed in the project. In that case, the metabolite was renamed but not the name of metabolite in the parent compound . Absolute edge case that probably never happens. Now that we import multiple compound at the same time, that code does not work and it would be quite hard to find a logic that make sense. => REMOVED